components/darkTheme/switchMode.tsx

```diff
@@ -0,0 +1,21 @@
+'use client'
+import React from 'react'
+import { useTheme } from 'next-themes'
+
+// Theme icons
+import DarkModeIcon from '@mui/icons-material/DarkMode'; // dark theme
+import WbSunnyIcon from '@mui/icons-material/WbSunny';  // light theme
+
```

Comment:
- Use react-icons library.
- We are now using Lucid icons, check Figma design.

Reply:
I will do it, I forgot to replace them

Reply:
Remove `@mui/*` packages as well.